studio/components/layouts/AppLayout/EnableBranchingButton/EnableBranchingModal.tsx

```diff
@@ -80,88 +90,129 @@ const EnableBranchingModal = () => {
           <IconGitBranch strokeWidth={2} size={20} />
           <div>
             <p className="text">Enable database branching</p>
-            <p className="text-sm text-light">Management environments in Supabase</p>
+            <p className="text-sm text-light">Manage environments in Supabase</p>
           </div>
         </Modal.Content>
 
-        {isLoadingIntegrations && (
+        {!isOrgBilling ? (
           <>
-            <Modal.Separator />
-            <Modal.Content className="px-7 py-6">
-              <GenericSkeletonLoader />
+            <Modal.Content className="px-3 border-y bg-alternative">
+              <Alert_Shadcn_ variant="default" className="rounded-none border-0">
+                <IconAlertCircle strokeWidth={2} />
+                <AlertTitle_Shadcn_>
+                  You will need to migrate to organization-based billing first
```

Comment:
need less technical language

```suggestion
                  Organization-based billing migration required
```